ansible_collections/arista/avd/molecule/eos_cli_config_gen_deprecated_vars/inventory/host_vars/host1/vlan-interfaces.yml

```diff
@@ -6,9 +6,6 @@ vlan_interfaces:
   Vlan1:
     description: test ipv6_address_virtual
     ipv6_enable: true
-    # Testing ipv6_address_virtual alone
-    # ipv6_address_virtual deprecated in 4.0.0. To be removed in 5.0.0
-    ipv6_address_virtual: fc00:10:10:1::1/64
 
   Vlan2:
     description: test ipv6_address_virtual and ipv6_address_virtuals
```

Comment:
change the description to what this tests is now doing

Reply:
updated